src/styles/utils/_key-farmes.scss

```diff
@@ -0,0 +1,24 @@
+@keyframes second {
+  0% {
+    transform: translateX(-50%) rotateZ(0deg);
+  }
+  100% {
+    transform: translateX(-50%) rotateZ(360deg);
+  }
+}
+@keyframes minute {
+  0% {
+    transform: translateX(-50%) rotateZ(0deg);
+  }
+  100% {
+    transform: translateX(-50%) rotateZ(360deg);
+  }
+}
+@keyframes hour {
+  0% {
+    transform: translateX(-50%) rotateZ(0deg);
+  }
+  100% {
+    transform: translateX(-50%) rotateZ(360deg);
+  }
+}
```

Comment:
this seems like totally the same animation, why not use one of it for all watch hands?